suite-common/message-system/scripts/validate-config.ts

```diff
@@ -1,26 +1,17 @@
 /* eslint-disable no-console */
-import Ajv from 'ajv';
-import * as fs from 'fs';
+import fs from 'fs';
 
-import { CONFIG_PATH, SCHEMA_PATH } from './constants';
-
-// checks that a config meets the criteria specified by the schema
-const validateConfigStructure = () => {
-    console.log('Validating config against schema...');
+import { validateJsonSchema } from '@trezor/node-utils';
 
-    const ajv = new Ajv();
+import { CONFIG_PATH, SCHEMA_PATH } from './constants';
 
+console.log('Validating config against schema...');
+try {
     const config = fs.readFileSync(CONFIG_PATH, 'utf-8');
     const schema = fs.readFileSync(SCHEMA_PATH, 'utf-8');
 
-    const validate = ajv.compile(JSON.parse(schema));
-    const isValid = validate(JSON.parse(config));
-
-    if (!isValid) {
-        throw Error(`Config is invalid: ${JSON.stringify(validate.errors)}`);
-    }
-
+    validateJsonSchema(config, schema);
     console.log('Config is valid!');
-};
-
-validateConfigStructure();
+} catch (error) {
+    console.error(error.message);
```

Comment:
btw wasn§t it supposed to throw before?

Reply:
Yes, you are right. I add it to rethrow here below.